test/test_common/environment.h

```diff
@@ -21,6 +22,17 @@ class TestEnvironment {
   static void initializeOptions(int argc, char** argv);
 
   /**
+   * @return bool if testing only with IP type addresses only.
```

Comment:
Nit: Please add a very short description of the method as you might expect in API documentation. I know it's redundant with the `@return` in these simple methods, but it will look less strange when rendered as a Doxygen description.